src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -291,6 +291,12 @@ TIP: If all the nodes of an AWS instance are removed, the instance will be termi
         This is an optional property. When it's not specified, a default key pair will be created or reused in the given region of the deployment.
         If specified, the name of the key pair (`vmKeyPairName`) to which this private key belongs to must be specified as well.
 
+-   *vmType*: The AWS VM type required for each VM (e.g. t3.medium).
+            An optional parameter, if not provided, selecting a specific machineType will be ignored.
+            For more information see https://aws.amazon.com/ec2/instance-types/[AWS EC2 Instance Types]
```

Comment:
unfortunately only a few vm types are supported.

It would be better to provide a link to jcloud javadoc where we see the list of vm types actually supported on EC2

Reply:
True, I changed accordingly.